Modules/CodeTrek/Resources/views/modals/sidebar-feedback-modal.blade.php

```diff
@@ -0,0 +1,86 @@
+<div class="feedback-modal modal fade" id="candidatefeedback{{ $codeTrekApplicant->id }}" tabindex="-1" z-index="1"
+    role="dialog" aria-labelledby="candidatefeedback{{ $codeTrekApplicant->id }}" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+        <form action="{{ route('codetrek.storeCodeTrekApplicantFeedback') }}" method="POST" id='feedback_form'>
+            @csrf
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel"><strong>Feedback Form</strong></h5>
```

Comment:
Please use meaningful name for id and if its not being used then remove it
```suggestion
                    <h5 class="modal-title" id="globalCodetrekFeedbackModal"><strong>Feedback Form</strong></h5>
```

Reply:
not needed, removed.